README.md

```diff
@@ -69,11 +69,15 @@ You should now be able to run Cylc commands, e.g.:
 
 Start the Hub (JupyterHub gets installed with the "cylc" package):
 
-- `jupyterhub
-    --JupyterHub.spawner_class="jupyterhub.spawner.LocalProcessSpawner"
-    --Spawner.args="['-s', '${CONDA_PREFIX}/work/cylc-ui']"
+```sh
+mkdir -p "${HOME}/srv/cylc/"
```

Comment:
```suggestion
mkdir -p "${HOME}/srv/cylc/"  # jupyterhub will write several files here to record your session info
```

Reply:
Seems fine, just suggesting a comment to inform users why we need this directory.

Reply:
Taking smile emoji as agreement :grimacing:

Reply:
(Ah, I don't have permission to commit to this repo, e.g. via the "suggestion" interface).

Reply:
Merging without comment for now (it's not critical, obviously), so I can announce the release.